Source/WebKit/UIProcess/Cocoa/BrowsingWarningCocoa.mm

```diff
@@ -116,6 +116,18 @@ static void addLinkAndReplace(NSMutableAttributedString *string, NSString *toRep
     return WEB_UI_NSSTRING(@"Website With Harmful Software Warning", "Unwanted software warning title");
 }
 
+static NSString *browsingWarningTitleText(BrowsingWarning::WarningType type)
```

Comment:
This isn't a great abstraction.  It takes a type, but seems like it should only be called with WarningType::HTTPSNavigationFailure.